src/core_plugins/elasticsearch/lib/migrate_config.js

```diff
@@ -1,24 +1,18 @@
-import { get } from 'lodash';
 import upgrade from './upgrade_config';
+import { SavedObjectsClient } from '../../../server/saved_objects';
 
-export default function (server, { mappings }) {
+export default async function (server, { mappings }) {
```

Comment:
Since server is just used to create the savedObjectsClient, how about updating these to simply accept the savedObjectsClient and we create it once in the health check?